tests/Generator.Tests/UnitTests.cs

```diff
@@ -746,6 +748,64 @@ public static async Task CallWithIProgressAsync()
 }
 """.Verify(false, true);
 
+    [Theory]

```

Comment:
Could you move all  IProgress related tests to IProgressTests.cs? This file is kinda big.